src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -729,6 +730,252 @@ public async Task ValidateInheritanceOfReadPermissionFromAnonymous()
             }
         }
 
+        [TestMethod]
+        public async Task CanPatchItemWithoutVariables()
+        {
+            // Run mutation Add planet;
+            string id = Guid.NewGuid().ToString();
+            const string name = "test_name";
+            string mutation = $@"
+mutation {{
+    createPlanet (item: {{ id: ""{id}"", name: ""{name}"" }}) {{
+        id
+        name
+    }}
+}}";
+            _ = await ExecuteGraphQLRequestAsync("createPlanet", mutation, variables: new());
+
+            const string newName = "new_name";
+            mutation = $@"
+mutation {{
+    patchPlanet (id: ""{id}"", _partitionKeyValue: ""{id}"", item: {{name: ""{newName}"", stars: [{{ id: ""{id}"" }}] }}) {{
+        id
+        name
+    }}
+}}";
+            JsonElement response = await ExecuteGraphQLRequestAsync("patchPlanet", mutation, variables: new());
+
+            // Validate results
+            Assert.AreEqual(newName, response.GetProperty("name").GetString());
+            Assert.AreNotEqual(name, response.GetProperty("name").GetString());
```

Comment:
what do you mean by "test execution would never get to this assert'?

Reply:
If code execution ever gets to the assert on line 760, the assert will never fail. Its purpose is redundant. 
```csharp
759:: Assert.AreEqual(newName, response.GetProperty("name").GetString());
760:: Assert.AreNotEqual(name, response.GetProperty("name").GetString());
```
If L759 assert passes, then that implies the patch was successful and it's impossible that L760 would fail. If `name` ==, `response.GetProperty("name").GetString()`, then L759 would always fail.

Reply:
got it, I have removed the other assertion and added more assertions.